src/index.js

```diff
@@ -3,12 +3,17 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from "react-router-dom";
 import { SnackbarProvider } from 'notistack';
 
+import { ThemeProvider } from '@material-ui/styles';
+import AppTheme from './AppTheme.js';
+
 import './index.css';
 import App from './App';
 
 ReactDOM.render(
     <BrowserRouter>
-        <SnackbarProvider maxSnack={3}>
-            <App />
-        </SnackbarProvider>
+        <ThemeProvider theme={AppTheme}>
```

Comment:
Disregard the comments on a `ThemeProvider` made previously 🤦‍♂

Reply:
:P